### PR TITLE
Switch to Meson

### PR DESCRIPTION
**Pull request by codygarver**

### CODE REVIEW

**Comment by codygarver** [comment]
I have ordered these commits as intended for master, please push them to master instead of squash merging.

**Comment by codygarver** [comment]
Closes #18

**Comment by codygarver** [comment]
Closes #3

File: README.md [hunk 1]
```diff
@@ -4,22 +4,18 @@
...
 
-    cmake -DCMAKE_INSTALL_PREFIX=/usr ..
-    make
-    
-To install, use `make install`, then execute with `cerbere
+To install, use `ninja install`, then execute with `cerbere`
 
-    sudo make install
+    sudo ninja install
     cerbere
```
Comment (danirabbit) [inline README.md]: Missed one for RDNN :)

**Review by danirabbit — CHANGES_REQUESTED** [review]
It looks like you're missing the post install for compiling schemas and you missed RDNNing the binary name in the README. Otherwise this compiles and appears to install things to places.

**Review by danirabbit — APPROVED** [review]
LGTM

**Pull request merged**